cmd/utils/flags.go

```diff
@@ -758,25 +761,57 @@ func SetupNetwork(ctx *cli.Context) {
 		core.BlockReward = big.NewInt(1.5e+18)
 		core.ExpDiffPeriod = big.NewInt(math.MaxInt64)
 	}
+	params.TargetGasLimit = common.String2Big(ctx.GlobalString(TargetGasLimitFlag.Name))
 }
 
-// SetupVM configured the VM package's global settings
-func SetupVM(ctx *cli.Context) {
-	vm.EnableJit = ctx.GlobalBool(VMEnableJitFlag.Name)
-	vm.ForceJit = ctx.GlobalBool(VMForceJitFlag.Name)
-	vm.SetJITCacheSize(ctx.GlobalInt(VMJitCacheFlag.Name))
+// MakeChainConfig reads the chain configuration from the given database or returns
+// an error if it failed.
+func MakeChainConfig(ctx *cli.Context) *core.ChainConfig {
```

Comment:
Please see https://github.com/ethereum/go-ethereum/pull/2281#discussion_r57310690

Also it doesn't return an error (opposed to what the comment says).